.github/workflows/build.yml

```diff
@@ -45,3 +45,16 @@ jobs:
       - name: Test
         run: |
           cmake --build build --target gc-check
+
+      - name: Build and install benchgc
+        working-directory: build
+        run: |
+          touch python_packages/gc_mlir_core/gc_mlir/extras/__init__.py
```

Comment:
for the workaround here,  the root casue is the mlir upstream did not adding this,  and maybe i can create a fix PR in upstream.

Reply:
We can remove this after your upstream PR got merged

Reply:
`find_namespace_packages` resolve this problem